pkg/client/clientset/versioned/typed/build/v1alpha1/application.go

```diff
@@ -63,7 +63,7 @@ func newApplications(c *BuildV1alpha1Client, namespace string) *applications {
 	}
 }
 
-// Get takes name of the application, and returns the corresponding application object, and an error if there is any.
+// Build takes name of the application, and returns the corresponding application object, and an error if there is any.
```

Comment:
where did this change come from?

Reply:
Oops - I clearly failed to dampen Goland's enthusiasm during one of the refactorings. Fixed.